### PR TITLE
Late breaking MAG region support update

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@e-tienne PTAL

**Comment by codyhoag** [comment]
@vikram-redhat let me know what you think about this one for 4.6 and whether it should wait until after docs freeze. QE would like to pull support for a couple of the listed MAG regions because they have not been fully tested yet.

File: modules/installation-azure-regions.adoc [hunk 1]
```diff
@@ -53,9 +53,11 @@ The installation program dynamically generates the list of available Microsoft A
...
-* usgovarizona (US Gov Arizona)
-* usgoviowa (US Gov Iowa)
 * usgovtexas (US Gov Texas)
 * usgovvirginia (US Gov Virginia)
+//* usdodcentral (US DoD Central)
+//* usdodeast (US DoD East)
+//* usgovarizona (US Gov Arizona)
+//* usgoviowa (US Gov Iowa)
+
+You can reference all available MAG regions in the link:https://azure.microsoft.com/en-us/global-infrastructure/geographies/#geographies[Azure documentation]. Other provided MAG regions are expected to work with {product-title}, but have not been tested.
```
Comment (e-tienne) [inline modules/installation-azure-regions.adoc:63]: LGTM. I was only able to confirm usgovtexas and usgovvirginia with our current access, although all regions are expected to work.

**Comment by vikram-redhat** [comment]
Definitely merge if these regions are not tested yet.

**Review by vikram-redhat — APPROVED** [review]

**Review by sfortner-RH — APPROVED** [review]
LGTM!

**Pull request merged**